languages/csharp/config.json

```
@@ -135,6 +135,12 @@
         "concepts": ["strings"],
         "prerequisites": ["basics"]
       },
+      {
+        "slug": "tuples",
+        "uuid": "8bb88ab5-3cde-4ebd-8eef-03323ebfd700",
+        "concepts": ["tuples"],
+        "prerequisites": ["basics", "strings", "conditionals"]
```

The `basics` prerequisite can be omitted as there are other prerequisites (see [this section](https://github.com/exercism/v3/blob/master/docs/maintainers/determining-concepts.md#basics)).

As per @robkeim's suggestion, we've also sorted the concepts and prerequisites by alphabet to not give the impression that one is more important than the other (which is hard to figure out).

```suggestion
        "prerequisites": ["conditionals", "strings"]
```